Laura/index.js

```diff
@@ -0,0 +1,34 @@
+// 13-Programming Basics: string methods
+
+// 1. Declare a variable with the value of "Hello World". Covert the value to upper case and print the converted value to the console.
+let str = "Hello World";
+console.log(str.toUpperCase()); // HELLO WORLD
+
+// 2. Declare another variable with the value of "Hello Earthling". Convert the value to lower case and print the converted value to the console.
+let str2 = "Hello Earthling";
+console.log(str2.toLowerCase()); // hello earthling
+
+// 3. Create a new string from a given string with the first character of the given string added at the front and back, i.e., expected output: Bananas => BBananasB.
+let str3 = "Bananas";
+// let str4 = str3[0] + str3 + str3[0];
+// console.log(str4);
+// let str4 = str3[0].concat(str3).concat(str3[0]);
+// console.log(str4);
+console.log(
+  str3
+    .replace(str3[0], str3[0].concat(str3[0]))
+    .replace(str3[str3.length - 1], str3[str3.length - 1].concat(str3[0]))
+);
+
+// 4. Create a new string from a given string taking the last 3 characters of the string and add them to both the front and back of the new string. The string length must be 3 or more, i.e., expected output: Scritch => tchScritchtch.
+let str4 = "Scritch";
+console.log(str4.substring(4) + str4 + str4.substring(4)); // tchScritchtch
+
+// 5. Create a new string from a given string by changing the position of first and last characters. The string length must be greater than or equal to 2, i.e., expected output: BoogieWoogie => eoogieWoogiB.
```

Comment:
The answer for task 5 is wrong! So I send you 2 solutions :
const boogieNightsStr = "BoogieWoogie";
const firstLetter = boogieNightsStr[0];
const lastLetter = boogieNightsStr[boogieNightsStr.length - 1];
const remainderString = boogieNightsStr.substring(1, boogieNightsStr.length - 1);
console.log(lastLetter + remainderString + firstLetter);
//or
let LastLetter = givenString.slice(-1);
let FirstLetter = givenString.substring(0, 1);
let switchedCharactersString = LastLetter + givenString.slice(1, -1) + FirstLetter;
console.log(switchedCharactersString);